app/src/main/java/nl/tudelft/b_b_w/model/Block.java

```diff
@@ -21,16 +22,18 @@
      * @param _ownHash            our own hash
      * @param _previousHashChain  the hash value of the block before in the chain
      * @param _previousHashSender the hash value of the block before of the sender
-     * @param _publicKey          public key of the owner of the block
+     * @param _publicKey          public key of the sender
+     * @param _iban               IBAN number of a contact
      * @param _isRevoked          boolean to check whether a block is revoked or not
      */
-    public Block(String _owner, int _sequenceNumber, String _ownHash, String _previousHashChain, String _previousHashSender, String _publicKey, boolean _isRevoked) {
+    public Block(String _owner, int _sequenceNumber, String _ownHash, String _previousHashChain, String _previousHashSender, String _publicKey, String _iban, boolean _isRevoked) {
```

Comment:
You should not be able to give a sequencenumber to the constructor upon creating a block